images/access-setup/content/bin/common.sh

```diff
@@ -37,8 +37,19 @@ get_context() {
     printf "[ERROR] Install jq\n" >&2
     exit 1
   fi
-  token_secret="$(KUBECONFIG="$target" kubectl get sa "$sa" -n "$namespace" -o json |
-    jq -r '.secrets[].name | select(. | test(".*token.*"))')"
+
+  token_secret=""
+  i=0
+  while [ -z "$token_secret" ]; do
+  	i=$((i+1))
+  	if [ $i -gt 10 ]; then
+      printf "Failed to extract secret token\n"
```

Comment:
Would it be a good idea to use this function [exit_error()](https://github.com/openshift-pipelines/pipeline-service/blob/main/images/access-setup/content/bin/common.sh#L17)

Reply:
> Would it be a good idea to use this function [exit_error()](https://github.com/openshift-pipelines/pipeline-service/blob/main/images/access-setup/content/bin/common.sh#L17)

@ramessesii2 the function [exit_error()](https://github.com/openshift-pipelines/pipeline-service/blob/main/images/access-setup/content/bin/common.sh#L17)  has a function `usage` inside it, maybe is not good to use it.

Reply:
ohh, missed that. thanks